codalab/rest/block_schemas.py

```diff
@@ -0,0 +1,96 @@
+"""
+Marshmallow schemas for serializing resource dicts into JSON API documents, and vice-versa.
```

Comment:
Should say something about which schemas (worksheet blocks).

Naming: at the top-level, should be specific and use `worksheet_block` (which goes for the file name).